control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -319,27 +338,33 @@ func (r *Reconciler) finalizeKind(ctx context.Context, broker *eventing.Broker)
 	}
 
 	topicConfig, brokerConfig, err := r.topicConfig(logger, broker)
```

Comment:
due to the `r.removeFinalizerCM(...)` call inside the `finalizeKind`, I keep the `topicConfig()` call outside of the "is not external topic" block...